.github/workflows/coding-standards.yml

```diff
@@ -86,10 +85,12 @@ jobs:
           path: .cache/phpcs.json
           key: ${{ runner.os }}-date-${{ steps.get-date.outputs.date }}-phpcs-cache-${{ hashFiles('**/composer.json', 'phpcs.xml.dist') }}
 
+      # Since Composer dependencies are installed using `composer update` and no lock file is in version control,
+      # passing a custom cache suffix ensures that the cache is flushed at least once per week.
       - name: Install Composer dependencies
         uses: ramsey/composer-install@83af392bf5f031813d25e6fe4cd626cdba9a2df6 # v2.2.0
         with:
-          composer-options: "--no-progress --no-ansi"
+          custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
```

Comment:
Considering the date is still retrieved in a separate step anyway for the PHPCS cache, why not re-use it ?

```suggestion
          custom-cache-suffix: ${{ steps.get-date.outputs.date }}
```

As a side-note: what with the two PHPCS runs in this workflow and these not using separate PHPCS caches, the caching of the PHPCS cache file is kind of useles....

Reply:
Would you mind opening the issues for it ? I can send in a PR to fix it. We'd only need to use different cache file names to get the optimal benefit in CI and that can be sorted via CLI arguments.